src/main/resources/assets/iris/lang/fr_fr.json

```diff
@@ -1,23 +1,28 @@
 {
   "iris.shaders.reloaded": "Shaders rechargés !",
+  "iris.shaders.toggled": "Shaders basculés sur %s !",
   "iris.keybind.reload": "Recharger les shaders",
+  "iris.keybind.shaderPackSelection": "Écran de sélection du pack de Shader",
+  "iris.keybind.toggleShaders": "Activation des Shaders",
+  "iris.keybinds": "Iris",
   "iris.shaders.reloaded.failure": "Échec lors du rechargement des shaders ! Raison : %s",
+  "iris.shaders.toggled.failure": "Échec lors de l'activation des shaders ! Raison : %s",
 
   "options.iris.apply": "Appliquer",
   "options.iris.refresh": "Rafraichir",
   "options.iris.openShaderPackFolder": "Ouvrir le dossier des shader packs...",
   "options.iris.refreshShaderPacks": "Rafraichir les packs de shader",
-  "options.iris.shaderPackSelection": "Shader Packs...",
-  "options.iris.shaderPackSelection.title": "Shader Packs",
+  "options.iris.shaderPackSelection": "Packs de Shaders...",
+  "options.iris.shaderPackSelection.title": "Packs de Shaders",
   "options.iris.shaderPackSelection.addedPack": "Shader pack ajouté : \"%s\"",
   "options.iris.shaderPackSelection.addedPacks": "%s shader packs ajoutés",
-  "options.iris.shaderPackSelection.failedAdd": "Les fichiers ne sont pas des shader packs valides",
+  "options.iris.shaderPackSelection.failedAdd": "Les fichiers ne sont pas des shaders packs valides",
   "options.iris.shaderPackSelection.failedAddSingle": "\"%s\" n'est pas un shader pack valide",
   "options.iris.shaderPackSelection.copyError": "Erreur lors de l'ajout du shader pack \"%s\"",
-  "options.iris.shaderPackSelection.copyErrorAlreadyExists": "\"%s\" est déjà dans votre dossier de shader packs !",
+  "options.iris.shaderPackSelection.copyErrorAlreadyExists": "\"%s\" est déjà dans votre dossier de packs de shaders !",
   "options.iris.shaders.disabled": "Shaders : Désactivés",
   "options.iris.shaders.enabled": "Shaders : Activés",
 
   "pack.iris.select.title": "Sélectionner",
-  "pack.iris.list.label": "+ Faites glisser vos shader packs pour les ajouter"
+  "pack.iris.list.label": "+ Faites glisser puis déposer vos packs de shaders pour les ajouter"
```

Comment:
It's a weird way of saying that this would be better
```suggestion
  "pack.iris.list.label": "+ Faites glisser vos packs de shaders ici pour les ajouter"
```

Reply:
I know but I checked the English version "pack.iris.list.label": "+ Drag and Drop Shader Packs to add" and it is basicly the famous "Glisser déposer" so idk if you care about keeping that parity with the English version or not.

Reply:
In that case it should be "Glissez déposez vos packs de shaders ici" because right now it looks like you have to do 2 things

Reply:
That's what we want yeah, thank you.